src/ansys/geometry/core/designer/component.py

```diff
@@ -395,6 +413,7 @@ def add_component(self, name: str, template: Optional["Component"] = None) -> "C
                         comp,
                         self._grpc_client,
                         template,
+                        instance_name,
```

Comment:
```suggestion
                        template=template,
                        instance_name=instance_name,
```
Any optional argument, please pass it with its identifier...